IABTechLab/uid2-self-serve-portal
# Email validation for adding team members and contacts

What Changed:
- Better validation for email when adding Team Member, adding Business Contact, adding new Participant
- Error message in dialog if user tries to enter a team member email that already exists for the given participant
- Updated error messages based on Gen's feedback

Test Plan:
- Enter valid and invalid emails for Add Team Member dialog, Add Business Contact dialog, Edit Business Contact Dialog, Add new participant in Contact Email field
- Add a team member that already exists
- Add a team member that exists in your portal db but not in the given participant and make sure that still gives a 400 error
